examples/flutter_gallery/lib/demo/shrine/shrine_home.dart

```diff
@@ -306,12 +304,8 @@ class _ShrineHomeState extends State<ShrineHome> {
   void showOrderPage(Product product) {
     final Order order = _shoppingCart[product] ?? new Order(product: product);
     final Completer<Order> completer = new Completer<Order>();
-    final Key productKey = new ObjectKey(product);
-    final Set<Key> mostValuableKeys = new HashSet<Key>();
-    mostValuableKeys.add(productKey);
     Navigator.push(context, new ShrineOrderRoute(
       order: order,
-      settings: new RouteSettings(mostValuableKeys: mostValuableKeys),
```

Comment:
I'd cargo-culted the code I'd originally written with most valuable keys out of confusion.I agree that although the feature makes sense it might be worth removing it in the interest of simplifying things.  It should be easy enough to restore if uses cases crop up.